### DISCUSSION TITLE
unable to get local issuer certificate

### DISCUSSION THREAD

**Discussion by NargiT** [community/community#34566] [Copilot Conversations]
Hello,

I am behind a corporate proxy with internal certificate. I am using vscode and intellij when I work and I recently subscribe to copilot.
I didn't have any issues with copilot plugin when using intellij, but with vscode extension I get this message error.

> GitHub Copilot could not connect to server. Extension activation failed: "unable to get local issuer certificate"

What can I do to ? 

EDIT: running on ubuntu 20.04 LTS and vscode was installed with `snap install code --classic`

**Comment by iamcjwilliams** [comment]
Hello @NargiT Can you try the following below? Here are steps for login from a clean state:

1. Close Visual Studio 2022.
2. Delete C:\Users<insert username>\AppData\Local\github-copilot\hosts.json
3. Log out of GitHub in the web browser.
4. Open Visual Studio.

**Reply by NargiT** [reply]
sorry I forgot to mention that I was on ubuntu and code was installed with `snap intall code --classic`
